Patches/MorrowRim/Projectiles.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+					<mods>
+						<li>Vanilla Expanded Framework</li>

```

Comment:
Being that the Vanilla Expanded Framework is far more common, it would probably make sense to check for it second, to save the patch checking for a ton of players that don't have MorrowRim in the first place.